inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_priorgridgenerator_node.cpp

```diff
@@ -42,11 +38,6 @@ MKLDNNExperimentalDetectronPriorGridGeneratorNode::MKLDNNExperimentalDetectronPr
     if (getOriginalInputsNumber() != 3 || getOriginalOutputsNumber() != 1)
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges!";
 
-    if (op->get_input_shape(INPUT_PRIORS).size() != 2 ||
```

Comment:
why removed at all? can we checked it via `getInputShapeAtPort`?

Reply:
nGraph operation has been validated already. As I know: we don't validate nGraph operation in other nodes. But, if you feel to keep validation here then I rollback changes.

Reply:
@maxnick, should we have validation inside CPU node, which has to be exist in nGraph operation?